app/helpers/checkout_helper.rb

```diff
@@ -11,6 +11,7 @@ def guest_checkout_allowed?
 
   def checkout_adjustments_for(order, opts = {})
     exclude = opts[:exclude] || {}
+    reject_zero_amount = opts[:reject_zero_amount] || true
```

Comment:
I think that this conditional can't become false.

Reply:
Good catch!! Thanks. 

Changed to 
```ruby
reject_zero_amount = opts[:reject_zero_amount].nil? ? true : opts[:reject_zero_amount]
```

Reply:
Another idea, I think `Hash#fetch` would be perfect here:

```
reject_zero_amount = opts.fetch(:reject_zero_amount, true)
```

Reply:
Oh, that would be beautiful. :sparkles:

Reply:
Done! 🙏 Thanks to you both!!